modules/core/include/opencv2/core/hal/intrin_sse.hpp

```diff
@@ -1329,9 +1330,25 @@ inline v_int8x16 v_absdiffs(const v_int8x16& a, const v_int8x16& b)
     v_int8x16 m = a < b;
     return (d ^ m) - m;
  }
-inline v_int16x8 v_absdiffs(const v_int16x8& a, const v_int16x8& b)
-{ return v_max(a, b) - v_min(a, b); }
+static inline v_int16x8 v_absdiffs(const v_int16x8& a, const v_int16x8& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
+
+static inline v_uint16x8 v_absdiffs(const v_uint16x8& a, const v_uint16x8& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
+
+static inline v_uint8x16 v_absdiffs(const v_uint8x16& a, const v_uint8x16& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
 
+static inline v_float32x4 v_absdiffs(const v_float32x4& a, const v_float32x4& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
```

Comment:
templates ?

Reply:
I can't apply templates because there are v_absdiffs() function with different implementation. For example:
`inline v_int8x32 v_absdiffs(const v_int8x32& a, const v_int8x32& b)
{
    v_int8x32 d = a - b;
    v_int8x32 m = a < b;
    return (d ^ m) - m;
}`